katana/static/app/factory/testcaseFactory.js

```diff
@@ -32,8 +32,15 @@ app.factory('TestcaseFactory', ['$http', '$routeParams', '$q', function($http, $
                     deferred.resolve(data);
                 })
                 .error(function(data, status, headers, config) {
-                    deferred.reject("Error while saving testcase xml: " + filename  + ' '
-                                    + status + ' ' + JSON.stringify(headers));
+                    sweetAlert({
+                        title: "Permission Denied.",
+                        closeOnConfirm: false,
+                        confirmButtonColor: '#3b3131',
+                        confirmButtonText: "Ok",
+                        text: "Please edit the directory permissions so that Katana can save this file.",
+                        type: "warning"
+                });
```

Comment:
Hi @priyaarun23,

Please add this check to every controller - suite, project, idf, and testdata